vite.config.ts

```diff
@@ -3,7 +3,7 @@ import reactRefresh from '@vitejs/plugin-react-refresh';
 
 // https://vitejs.dev/config/
 export default defineConfig({
-    base: '/app/',
+    base: '/',
```

Comment:
I realized this change makes the _demo_ of the chromoscope inaccessible via `chromoscope.bio/app`. I tested this by running the following commands. 

```sh
# build both the documentation and demo and create a "\build" folder to organize files
yarn predeploy

# enables browsing the built project (i.e., doc and demo) by running a local server
http-server ./build
```

How the document and demo are organized under `\build`:

```
build
   - index.html // ← documentation (i.e., "chromoscope.bio")
   - ...
   - app
      - index.html // ← demo (i.e., "chromoscope.bio/app")
      - ...
```

By setting `base: '/app/'`, the `index.html` file for the demo (i.e., `app/index.html`) created by `vite` has proper relative paths (e.g., `app/index.js` instead of `index.js`). So, without setting the `base` option, `chromoscope.bio/app` (`app/index.html`) ends up having the wrong relative paths, making the demo not work.

I did not find a simple way to walk around this and would need to experiment more for a better alternative, but we can simply create an inlined HTML file for the webpage that you created (instead of using React) and put that under a `/dev/codec/` folder. 

```
build
   - index.html // ← documentation (i.e., "chromoscope.bio")
   - ...
   - app
      - index.html // ← demo (i.e., "chromoscope.bio/app")
      - ...
   - dev
      - codec
         - index.html // ← simple page to convert JSON to str
```

In this way, users should be able to access the webpage via "chromoscope.bio/dev/codec/'.